### PR TITLE
Podman Machine 5 Refactor

### PR DESCRIPTION
**Pull request by baude**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -387,58 +387,58 @@ alt_build_task:
...
-            }
-        }
-        arc unarchive repo.tbz2 .\
-        if ($LASTEXITCODE -ne 0) {
-            throw "Unarchive repo.tbz2 failed"
-            Exit 1
-        }
-        Get-ChildItem -Path .\repo
-    main_script: ".\\repo\\contrib\\cirrus\\win-installer-main.ps1"
+      #win_installer_task:
```
Comment (rhatdan) [inline .cirrus.yml:390]: I don't think these should be commented out?

Comment (baude) [reply .cirrus.yml:390]: no they should

**Comment by mheon** [comment]
@baude Needs a rebase.

**Review by ashley-cui — COMMENTED** [review]
LGTM

File: .cirrus.yml [hunk 2]
```diff
@@ -489,35 +489,35 @@ osx_alt_build_task:
...
-        image_family: freebsd-13-2
-    setup_script:
-        - pkg install -y gpgme bash go-md2man gmake gsed gnugrep go pkgconf
-    build_amd64_script:
-        - gmake podman-release
-    # This task cannot make use of the shared repo.tbz artifact and must
-    # produce a new repo.tbz artifact for consumption by 'artifacts' task.
-    repo_prep_script: *repo_prep
-    repo_artifacts: *repo_artifacts
+#freebsd_alt_build_task:
```
Comment (Luap99) [inline .cirrus.yml:492]: Does this break the freebsd build or why is this commented? I wouldn't block on it right now but we should fix the freebsd build before the final release.

Comment (baude) [reply .cirrus.yml:492]: freebsd does not build rn ... its on my list of first things to kind of do when this merges

**Review by Luap99 — APPROVED** [review]
/lgtm
/hold

**Pull request merged**